src/modules/cpu/kernel/glitch.hpp

```diff
@@ -0,0 +1,1133 @@
+#include <x86intrin.h>
+#include <smmintrin.h>
+#include <immintrin.h>
+#include <fstream>
+#include <cstring>
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+
+template <typename T> void convert_f32_Pln3_Pkd3(T *srcPtrChannel, T *dstPtrChannel, RpptDescPtr srcDescPtr, RpptDescPtr dstDescPtr, int width)
+{
+    T *srcPtrRowR, *srcPtrRowG, *srcPtrRowB, *dstPtrRow;
+    srcPtrRowR = srcPtrChannel;
+    srcPtrRowG = srcPtrRowR + srcDescPtr->strides.cStride;
+    srcPtrRowB = srcPtrRowG + srcDescPtr->strides.cStride;
+    dstPtrRow = dstPtrChannel;
+    for(int i = 0; i < srcDescPtr->h ; i++)
+    {
+        T *srcPtrTempR, *srcPtrTempG, *srcPtrTempB, *dstPtrTemp;
+        srcPtrTempR = srcPtrRowR;
+        srcPtrTempG = srcPtrRowG;
+        srcPtrTempB = srcPtrRowB;
+        dstPtrTemp = dstPtrRow;
+
+        int vectorLoopCount = 0;
+        for (; vectorLoopCount < width; vectorLoopCount += 8)
+        {
+            if( typeid(T) == typeid(Rpp32f))
```

Comment:
Please remove space before typeid(T)
Applicable to all such instances in this file

Reply:
Done